src/dispatch-payload.ts

```diff
@@ -16,7 +22,7 @@ import {
 
 // Given a function callback_id, returns a set of potential function files to check
 type GetFunctionFilesCallback = {
-  (functionCallbackId: string): string[];
+  (functionCallbackId: string): (string | FunctionModule)[];
```

Comment:
Allowing for tests to provide function module objects instead of having to use real files on the file system. This makes it a lot easier to test and mock different variations of functions module definitions.

Reply:
Love this, great idea